masterlist.yaml

```
@@ -19570,8 +21142,11 @@ plugins:
     inc: [ 'Duke Patricks - Basic Combat Geometry.esp' ]
     msg:
       - type: say
-        content: 'Requires: Underdark: The Realm of Northdark.'
-  - name: 'GG_Valeria_Revived.esp'
+        content:
+          - lang: en
+            text: 'Requires: Underdark: The Realm of Northdark.'
+          - lang: de
+            text: 'Benötigt: Underdark: The Realm of Northdark.'
```

```suggestion
            text: 'Benötigt: Underdark: The Realm of Northdark.'
  - name: 'GG_Valeria_Revived.esp'
```
Deleted this line by accident.